tests/TestCase/Http/ServerRequestTest.php

```diff
@@ -3543,7 +3543,7 @@ public function testGetEnv()
         $request = new ServerRequest();
 
         //Test default null
-        $this->assertSame($request->getEnv('Foo'), null);
+        $this->assertNull($request->getEnv('Foo'), null);
 
         //Test default set
         $this->assertSame($request->getEnv('Foo', 'Bar'), 'Bar');
```

Comment:
Those are still using the wrong argument order. "expected, result" is what it should be.